config/emailServiceConfig.yaml

```diff
@@ -0,0 +1,2 @@
+host: "email"

```

Comment:
I think this works for now, but I imagine in the future we're going to need to make these based in environment variables, because this won't fit our needs once we're deployed.